cluster_agent/jobbergate/submit.py

```diff
@@ -86,10 +83,11 @@ async def submit_job_script(
             or SETTINGS.DEFAULT_SLURM_WORK_DIR
         )
 
-        local_script_path = submit_dir / f"{name}.job"
-        local_script_path.write_text(job_script)
-
-    logger.debug(f"Copied job_script to local file {local_script_path}.")
+        for path, file_content in pending_job_submission.job_script_files.files.items():
+            local_script_path = submit_dir / path
```

Comment:
what is path in this context? Can you give an example of how this local script path looks like?

Reply:
It is the file path, like `jobbergate.py`, or `job-script.sh`, or `./input/parameters.json`